src/downloader.jl

```diff
@@ -30,7 +30,7 @@ struct PathDownloader <: AbstractDownloader
     lru::LRU{String, Int}
 end
 
-function PathDownloader(cache_dir; timeout=10, cache_size_gb=5)
+function PathDownloader(cache_dir; timeout=10, cache_size_gb=50)
```

Comment:
That seems a little extreme...maybe 25?

Reply:
yeah, if is actually GB, I would not like to go  beyond 1GB. Which is the default we use in DiskArrays.jl

Reply:
Disk space is cheap ^^ Well, I guess we can go back to 5gb... 1gb sounds really small though for pointcloud tiles.
That's like 5 bigger tiles ^^

Reply:
I mean I don't have 50gb free disk space 🤣

Reply:
Me neither🥲